src/utils/runtime/src/lib.rs

```diff
@@ -156,8 +158,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
             .with_target("aws_sdk_ec2", Level::INFO)
             .with_target("aws_sdk_s3", Level::INFO)
             .with_target("aws_config", Level::WARN)
-            .with_target("aws_smithy_types", Level::INFO)
-            .with_target("aws_credential_types", LevelFilter::INFO)
+            .with_target("aws_smithy_client", Level::DEBUG)
```

Comment:
I think we only need to enable debug logs for `aws_smithy_client::retry` instead of the whole crate. Not sure whether this will cause excessive logs printing.